pkg/secrets/secrets_test.go

```diff
@@ -0,0 +1,37 @@
+package secrets
+
+import (
+	"testing"
+
+	oconfig "github.com/openshift/api/config/v1"
+)
+
+func Test_processTags(t *testing.T) {
+	type args struct {
+		platformType oconfig.PlatformType
+		userData     string
+	}
```

Comment:
I dont see the need to create this struct

Reply:
Removed.